libtest/unittest.cc

```diff
@@ -1069,8 +1069,8 @@ static test_return_t clear_servers(void* object)
 
 static test_return_t check_for_memcached(void* object)
 {
-#if defined(HAVE_CYASSL) && HAVE_CYASSL
-  SKIP_IF(HAVE_CYASSL);
+#if defined(HAVE_WOLFSSL) && HAVE_WOLFSSL
+  SKIP_IF(HAVE_WOLFSSL);
```

Comment:
This skips the test when using WolfSSL, but it doesn't skip the test when using OpenSSL. Do you know if there's a reason for that?